src/Alerts/TranslationKeyNotFound.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types=1);
 namespace Airship\Alerts;
 
 /**
```

Comment:
(To explain the joke: I added `strict_types=1` to the files defining custom `Exception` classes.)